doc/fropin.xml

```diff
@@ -28,3 +37,173 @@ false
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="AsListCanonical">
+  <ManSection>
+    <Attr Name="AsListCanonical" Arg="S"/> 
+    <Attr Name="EnumeratorCanonical" Arg="S"/>
+    <Attr Name="IteratorCanonical" Arg="S"/>
+    <Returns>A list, enumerator, or iterator.</Returns>
+    <Description>
+      When the argument <A>S</A> is a semigroup in the representation 
+      <Ref Filt="IsEnumerableSemigroupRep"/>, <C>AsListCanonical</C>
+      returns a list of elements of <A>S</A> in the order they are enumerated
```

Comment:
I recommend adding "the": `a list of the elements` - just to make it clear that it contains all elements.